bwa/0.7.17/Dockerfile

```diff
@@ -0,0 +1,27 @@
+################## BASE IMAGE ######################
+FROM biocontainers/biocontainers:v1.1.0_cv2
+
+################## METADATA ######################
+LABEL base_image="biocontainers:v1.1.0_cv2"
+LABEL version = "5"
```

Comment:
no space between label and value (version=5)

by the way, should be version 1 as it is first version of the release

version is internal counter, not related to tool release. You only increase it (default to be set to 1) when modifying the Dockerfile for the same tool release

Reply:
Okay, thanks for pointing it out. I thought the version is used to track how many updates you have for the software.